code/modules/research/experimentor.dm

```diff
@@ -533,13 +533,8 @@
 		autoexperiment = 0
 		visible_message("<span class='warning'>[src] melts [exp_on], ian-izing the air around it!</span>")
 		throwSmoke(loc)
-		if(trackedIan)
-			throwSmoke(trackedIan.loc)
-			trackedIan.forceMove(loc)
-			investigate_log("Experimentor has stolen Ian!", INVESTIGATE_EXPERIMENTOR) //...if anyone ever fixes it...
-		else
-			new /mob/living/simple_animal/pet/dog/corgi(loc)
-			investigate_log("Experimentor has spawned a new corgi.", INVESTIGATE_EXPERIMENTOR)
+		new /mob/living/simple_animal/pet/dog/corgi(loc)
+		investigate_log("Experimentor has spawned a fake Ian.", INVESTIGATE_EXPERIMENTOR)
```

Comment:
```suggestion
		investigate_log("Experimentor has spawned a corgi.", INVESTIGATE_EXPERIMENTOR)
```
No longer Ian unless you add some code to rename the spawned corgi to Ian.

Reply:
An alternative would be to make this Ian's evil twin or something, like a Star Trek teleporter-accident episode.